documentation/decisions/001_styling_library.md

```diff
@@ -0,0 +1,40 @@
+---
+status: Accepted
+date: 2023-02-03
+deciders: EasyPost UI Platform
```

Comment:
Thoughts on calling out individuals? We could use github usernames

Reply:
yes agreed